### DISCUSSION TITLE
PR for clair report parsing

### DISCUSSION THREAD

**Discussion by u-ahmedofficial** [DefectDojo/django-DefectDojo#9203] [General]
Hi, i would like to submit a PR for clair report parsing which is not compatible with the new versions in 2023. I have made changes locally and those are working, i would like to contribute as well for future audience. However, i am unable to push the changes to bugfix branch.

**Comment by manuel-sommer** [comment]
Hi @u-ahmedofficial, 
did you create a fork of DefectDojo? Then you have to push to a new branch and make a PR from there to the original repo. You can also take a look here:
https://github.com/DefectDojo/django-DefectDojo/blob/master/readme-docs/CONTRIBUTING.md

**Reply by u-ahmedofficial** [reply]
What i tried was:

First Try:

1. Clone the repo
2. Create a new branch 
3. update the code, commit
4. Tried to push it was not possible

Second try:

1. Clone the repo
2. Switch to bugfix branch
3. Commit
4. Push fails

Sure, i can try with forking, as the instructions were not clear on forking

**Reply by manuel-sommer** [reply]
Yeah, it works with forking.

**Comment by manuel-sommer** [comment]
@u-ahmedofficial could you mark my above comment as the answer?

**Comment by u-ahmedofficial** [comment]
Thanks this process worked.
